rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -44,8 +44,25 @@ import com.normation.inventory.domain.NodeId
 
 
 /**
+ * The main class helping maps expected
+ * reports for a node
+ */
+case class RuleNodeExpectedReports(
```

Comment:
Ok, I understand the structure, that they are above the Node, and that expected reports are now completely "by node". 

I thinks it's the naming that confuses me ( and i would rename it RuleExpectedReports, but it still exists and it's goot that it creates a link with RuleNodeStatusReports) So maybe I'll be happier with a refactoring, but I think it's not the place to do it right now

The full datastrucuture is complicated with its 4 level map (there is one more level RuleSerialedId) And i fear it might be hard to maintain ( like in 3 months ;) ), but i have really NO idea of how to do it better.